resources/views/pages/user-shifts.html

```diff
@@ -60,14 +63,14 @@ <h1>%title% %add_link%</h1>
             </div>
         </div>
         <div class="col-md-6">
-            <button class="btn btn-info btn-sm mb-3 d-print-none" type="button"
+            <button class="btn btn-info mb-3 d-print-none" type="button"
```

Comment:
Can you tell me why you want this button to be bigger? Since this takes up more space on the page and moves the shifts even more down on the screen, this looks worse to ma especially using a small screen.

Reply:
ah, my reasoning was that the different button sizes add visual noise and a hierarchy that doesn't exist functionally (at least from my view), making the page harder to skim. I think once a user *knows* that there's a table below the long list, it's less critical if the list pushes the table down a bit further. I think in the long run, maybe the list could be made scrollable instead of displaying it all at once (similar to filter options in fashion online shops– they're really hard to screenshot because the scroll bar keeps disappearing obviously) to resolve the problem even better, but that's for another PR :D

Reply:
>  the list could be made scrollable instead of displaying it all at once 

orrrrr they could be moved to the side as an "overlay" and be made toggleable there?

![IMG_9830](https://github.com/user-attachments/assets/8feb5a7a-6ef6-419d-b650-a8d8016da64a)

but now I'm just throwing ideas around, that's all for a future PR.

Reply:
I like your ideas. But since we still have the long lists and there are people that have the filters expanded all the time, I would prefer the smaller button for now. Could you open an issue and move your ideas to that issue (if you want you can start the title with "[RFC]", which stands for "request for comments", so that more people can share their ideas and opinions)?

Reply:
I made these filter selections scrollable in my fixes PR #1538